cpu/efm32/periph/uart.c

```diff
@@ -164,10 +163,62 @@ void uart_write(uart_t dev, const uint8_t *data, size_t len)
 void uart_poweron(uart_t dev)
 {
     CMU_ClockEnable(uart_config[dev].cmu, true);
+
+    if (_is_usart(dev)) {
+        USART_TypeDef *usart = uart_config[dev].dev;
+
+        /* enable tx */
+        usart->CMD = USART_CMD_TXEN;
```

Comment:
Please use `USART_Enable` (and `LEUART_Enable`). There is [more happening](http://devtools.silabs.com/dl/documentation/doxygen/5.7/efr32mg12/html/em__usart_8c_source.html#l00663) in these methods to have compatibility with other EFM32 families.

Reply:
The emlib functions can't be used to enable TX and RX independently though right?

Reply:
Nevermind. They can of course.